src/executables/util_service.c

```diff
@@ -2540,34 +2560,44 @@ process_javasp_status (const char *db_name)
 }
 
 static int
-process_javasp (int command_type, int argc, const char **argv, bool process_window_service)
+process_javasp (int command_type, int argc, const char **argv, bool show_usage, bool process_window_service)
 {
   char buf[4096] = { 0 };
   char *list = NULL, *save = NULL;
   const char *delim = " ,:";
   int status = NO_ERROR;
   char *db_name = NULL;
+  int master_port = prm_get_master_port_id ();
 
   if (argc == 0)		/* cubrid service command */
     {
-      if (us_Property_map[SERVER_START_LIST].property_value != NULL)
+      /* get all server names from master request */
+      if (css_does_master_exist (master_port))
 	{
-	  strncpy (buf, us_Property_map[SERVER_START_LIST].property_value, sizeof (buf) - 1);
+	  get_server_names (buf);
```

Comment:
I think that get_server_names() function only can get list of servers which execute with heartbeat.
But, it seem that here need all executed server.
please check get_server_names() function.

Reply:
I've fixed to get all executed servers. By `cub_commdb -O`, HA-applylogdb and HA-copylogdb are also returned. So I've changed only to handle server names including HA-Server.